config/application.rb

```diff
@@ -30,20 +18,45 @@ class Application < Rails::Application
       #{config.root}/app/workers
     )
 
-    # Do not swallow errors in after_commit/after_rollback callbacks.
-    config.active_record.raise_in_transactional_callbacks = true
+    # Use Rails 7 zeitwerk eager loading
+    config.eager_load_paths += %W(
+      #{config.root}/app/models/validators
+      #{config.root}/app/representers/concerns
+      #{config.root}/app/workers
+    )
+
+    # # Do not swallow errors in after_commit/after_rollback callbacks.
+    # config.active_record.raise_in_transactional_callbacks = true
+
+    # Initialize configuration defaults for originally generated Rails version.
+    config.load_defaults 7.0
 
     config.active_job.queue_adapter = :shoryuken
     config.active_job.queue_name_prefix = ENV['ANNOUNCE_RESOURCE_PREFIX'] || ENV['RAILS_ENV']
     config.active_job.queue_name_delimiter = '_'
 
-    config.assets.enabled = false
-
-    config.cache_store = :memory_store, { size: 128.megabytes }
```

Comment:
Did we lose the cache store for staging/production?

Reply:
oops! fixed!

Reply:
I think I was planning on setting up a better cache in stag/prod, then just never did it, so I'll put this back.